src/main/java/core/basesyntax/Lottery.java

```diff
@@ -2,12 +2,12 @@
 
 import java.util.Random;
 
-public class Lottery extends Ball {
-    public String getRandomBall() {
-        int number;
+public class Lottery {
+    public Ball getRandomBall() {
+        final int max_number = 100;
```

Comment:
let's move this variable from method to class
and make it constant
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names